gflownet/gflownet.py

```diff
@@ -1081,7 +1101,7 @@ def train(self):
             # Log
             if self.logger.lightweight:
                 all_losses = all_losses[-100:]
-                all_visited = states_term
+                all_visited = states_term  # TODO-V: unused
```

Comment:
Yes unused because it is not maintained. But computing metrics from the visited state is a nice to-have. It should be done with the next version of the Buffer though, not here to not clutter too much the PR. 

So: can remove